spec/integration/_dev/benchmark/system.scenario.spec.yml

```diff
@@ -1,5 +1,15 @@
 spec:
   definitions:
+    input:
+      description: The input of the package to benchmark.
+      type: string
+    vars:
+      description: The package level variables.
+      type:
+        - object
+        - "null"
+      additionalProperties: true
```

Comment:
I haven't created the definition: I just moved it from properties :)

not sure we can be more specific: the `vars` properties depends on each package.

indeed touching this file is not required anymore, because I was able to get rid of `input` and `vars` in the rally spec, so I will just revert :)